rhv_checks.py

```diff
@@ -338,6 +339,43 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, warning = [], [], []
+    # Get all the hosts with details.
+    hosts = system.api.system_service().hosts_service().list(all_content=True)
+
+    for host in hosts:
+        engine_value = all((host.hosted_engine.active, host.hosted_engine.configured))
+        maintenance_value = any((host.hosted_engine.global_maintenance, host.hosted_engine.local_maintenance))
+        if not engine_value or maintenance_value:
+            critical.append((host.name, "False"))
+        else:
+            okay.append((host.name, "True"))
+        # Checing hosted engine non-zero scrore.
+        if host.hosted_engine.score == 3400:
+            okay.append((host.name, host.hosted_engine.score))
+        else:
+            warning.append((host.name, host.hosted_engine.score))
+
+    if critical:
+        msg = ("Critical: The following host's Hosted Engine-Status is definitely have an issue: {state}\n "
+              "Status of all host is: {state}".format(state=critical))
+        logger.error(msg)
+        print(msg)
+        sys.exit(2)
+    elif warning:
+        msg = ("Warning: The following host's Hosted Engine score reported below 3400".format(warning))
```

Comment:
I would change this to "Warning: the hosted engine score of the following hosts is below 3400: {warning}."